Assignment1 string

AadityaJain16 opened the pull request:
Added the files again to the assignment 1 branch

AadityaJain16 committed:
corrected code

remove while loop

AadityaJain16 committed:
Deleted Content from Child branch

AadityaJain16 committed:
Added the Assignment Code

Added code for assignment 1

Himanshucodeup commented on Main.java:
```diff
@@ -0,0 +1,64 @@
+
+        System.out.println("Enter your input string");
+        String input = sc.nextLine();
+        MyString s = new MyString();
```
sc and s are not descriptive names, make every variable names like these more descriptive. their names should tell exactly what they are being used for

Himanshucodeup commented on MyString.java:
```diff
@@ -0,0 +1,185 @@
+
+public class MyString {
+
+
```
unnecessary spaces

Himanshucodeup commented on MyString.java:
```diff
@@ -0,0 +1,185 @@
+
+    public int CountWord(String input)
+    {
+        String temp="";
```
put spaces around operators and spaces to the right of , and ; everywhere for more readability

Himanshucodeup commented on Main.java:
```diff
@@ -0,0 +1,64 @@
+        String input = sc.nextLine();
+        MyString s = new MyString();
+        
+            System.out.println("Choose the operation");
```
add a loop for multiple operations without having to restart

AadityaJain16 committed:
Refactor string operations and improve CLI flow

Updated Main.java to use a loop for multiple string operations, improved user prompts, and refactored MyString.java to reuse a Scanner instance for input. Renamed methods for consistency, improved code readability, and fixed minor logic issues in string manipulation methods.

AadityaJain16 committed:
Updated Assignment 1
